sql/src/main/java/io/crate/operation/projectors/BatchConsumerToRowReceiver.java

```diff
@@ -114,6 +117,7 @@ private void consumeIterator(BatchIterator iterator) {
             }
         } catch (Throwable e) {
             // catch Throwables to capture assertion errors
+            LOGGER.error(e.getMessage(), e);
```

Comment:
In the past we tried to avoid logging errors that are forwarded or propagated because it usually leads to errors appearing multiple times in the output which can be confusing.